src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -42,7 +42,7 @@ public void storeMessage(Message message) {
     messageEntity.setProperty("user", message.getUser());
     messageEntity.setProperty("text", message.getText());
     messageEntity.setProperty("timestamp", message.getTimestamp());
-
```

Comment:
You're dropping the empty line between the setProperty calls and the call to datastore.put. Is that intended?

Reply:
This was not something that was intended, do I add back in the new line?